### DISCUSSION TITLE
Why I cannot obtain the model graph like the tutorial?

### DISCUSSION THREAD

**Discussion by yufeng-tian** [lnccbrown/HSSM#592] [Q&A] [answered]
I use the code "ddm_model_only_v.graph()" and Jupyter lab generates this graph (see Picture 1). However, the graph is not the same as that in the tutorial (see Picture 2). Would you like me to know how to create the same figure?

![1](https://github.com/user-attachments/assets/b07fe99c-5ad1-42fd-a55f-03fa08a4b56b)
![2](https://github.com/user-attachments/assets/99c4459d-9766-4b81-850c-913fe0cc2ac1)

**Comment by digicosmos86** [comment, marked as answer]
Hi @yufeng-tian we switched to the new graph structure in HSSM v0.2.4, which is more streamlined. The graphs in this tutorial are deprecated and we will update that soon. Thanks for bringing this to our atttention!
